src/components/Navbar.jsx

```diff
@@ -95,15 +95,14 @@ const Navbar = () => {
             Get Hired
           </NavLink>
         </p>
-        <p className="bg-[#687eff] px-12 py-3 rounded-full">
-          <a
-            href="https://discord.gg/vee94km4Wh"
+        <p className="bg-[#687eff] px-12 py-3 rounded-full uppercase text-md font-semibold transition-all duration-200 ease-out text-white  hover:text-[black]">
```

Comment:
You change discord link to hire link. fix this

Reply:
Fixed this in #61